notebooks/notebooks/iclr2022/elo-by-visits-2.pgf

```diff
@@ -892,15 +892,15 @@
 \pgfusepath{stroke,fill}%
```

Comment:
this plot got regenerated because the experiment it depends on is still running and outputting new data